deployments/root/conda-envs/alchemiscale-fah-compute.yml

```diff
@@ -5,24 +5,28 @@ channels:
 dependencies:
   - pip
   - python =3.12
-  - cudatoolkit =11.8
+  - cuda-version >=12
 
-  - alchemiscale-compute=0.5.3
+  - alchemiscale-compute =0.6.0
   
   # openmm protocols
-  - feflow=0.1.1
+  - feflow =0.1.2
 
   # additional pins
-  - gufe=1.1.0
-  - openfe=1.2.0
-  - openmm=8.1.2
-  - openmmforcefields>=0.14.1
-  - openff-units=0.2.2
+  - gufe =1.2.0
+  - openfe =1.3.0
+  - openmm =8.1.2
+  - openmmforcefields >=0.14.1
+  - openff-units =0.2.2
+
+  ## pins due to breaking changes
+  - openff-interchange =0.4.1    # https://github.com/openmm/openmmforcefields/issues/365
 
   # alchemiscale-fah dependencies
   - cryptography
   - plyvel
   - zstandard
 
   - pip:
-    - git+https://github.com/openforcefield/alchemiscale-fah.git@v0.1.1
+    - git+https://github.com/openforcefield/alchemiscale-fah.git@v0.1.2
+    - git+https://github.com/OpenFreeEnergy/pontibus.git@v0.0.1
```

Comment:
I'm not sure exactly, but this won't be needed for F@H execution, so could drop pontibus here unless you need the compute environments to match.

Reply:
Yes, it shouldn't be necessary if our FAH compute services never touch `pontibus` protocols (which they shouldn't). Removing!